package.json

```diff
@@ -29,7 +29,7 @@
         "jest-ssr": "jest test/ssr --env=node --no-cache",
         "jest-e2e": "rm -f ./test/e2e/screenshots/*.png && jest test/e2e",
         "get-size": "gzip -9 < dist/size.min.js | wc -c",
-        "check-size": "npm run webpack-size && if [ $(npm run -s get-size) -ge 70000 ]; then echo 'Bundle size greater than 65k'; exit 1; else npm run delete-size; fi;",
+        "check-size": "npm run webpack-size && if [ $(npm run -s get-size) -ge 85000 ]; then echo 'Bundle size greater than 65k'; exit 1; else npm run delete-size; fi;",
```

Comment:
This is concerning. The script size jumped 15k?

Reply:
I tested this change locally and ran `npm run check-size` and it seemed to pass fine without changing this threshold. I got the following output:
```
Final bundle size: 66362
```